vlib/v/checker/tests/multi_return_arg_missing_err.vv:9:14: error: unknown function: ret_2
    7 | 
    8 | fn main() {
    9 |     expect_2(1, ret_2())
      |                 ~~~~~~~
   10 |     expect_2(returning_2(), 1)
   11 | }
vlib/v/checker/tests/multi_return_arg_missing_err.vv:9:14: error: `ret_2()` (no value) used as value in argument 2 to `expect_2`
    7 | 
    8 | fn main() {
    9 |     expect_2(1, ret_2())
      |                 ~~~~~~~
   10 |     expect_2(returning_2(), 1)
   11 | }
vlib/v/checker/tests/multi_return_arg_missing_err.vv:10:26: error: expected 2 arguments, but got 3
    8 | fn main() {
    9 |     expect_2(1, ret_2())
   10 |     expect_2(returning_2(), 1)
      |                             ^
   11 | }
Details: have ((int, int), int literal)
         want (int, int)
